app/views/layouts/_header.html.erb

```diff
@@ -2,7 +2,7 @@
   <div class="container">
     <div class="row">
       <div class="col-xs-6">
-        <%= link_to header_title, header_link, id: "logo" %>
+        <%= link_to "Points", root_url, id: "logo" %>
```

Comment:
This was lis this before but was changed by ernesto a few months ago, I would wait to first check with him about this https://github.com/fastruby/points/pull/92 and also nathalie offered to help with UX in general so I wouldn't change this now in this PR anyway

The issue this PR refers to (https://github.com/fastruby/points/issues/75) is focused mainly on subproject <-> subproject navigation, so changing the navbar to go back to the home shouldn't be needed (if we have to go back to the home to go to a sibling subproject then this is not addressing the issue)